cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -210,6 +214,56 @@ hunter_boost_component_b2_args(
 
 list(APPEND build_opts ${b2_component_opts})
 
+#
+# handle iostreams dependency on zlib and bzip2 from
+# within Hunter
+# TODO: DRY
```

Comment:
Actually what do you think about moving this module to `hunter_boost_component_b2_args`? It seems we already have all information inside.